Bitrise/ViewModel.swift

```diff
@@ -19,3 +19,4 @@ struct ViewModel: ViewModelType {
         quantity.value(String(value + 1))
     }
 }
+
```

Comment:
Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)